### PR TITLE
Include participants in projects csv export

### PR DESCRIPTION
**Pull request by gr-eg**
In the format `Participant 1 First Name`, `Participant 2 First Name` etc

### CODE REVIEW

File: server/routes/handlers/projects.js [hunk 1]
```diff
@@ -128,32 +128,48 @@ module.exports = {
...
+            { label: 'Owner Email', value: 'owner.email' },
+            { label: 'Created At', value: row => dateFormat(row.createdAt) },
+            { label: 'Updated At', value: row => dateFormat(row.updatedAt) },
+            { label: 'Supervisor First Name', value: 'supervisor.firstName' },
+            { label: 'Supervisor Last Name', value: 'supervisor.lastName' },
+            { label: 'Supervisor Email', value: 'supervisor.email' },
+            { label: 'Supervisor Phone', value: 'supervisor.phone' },
+          ];
+
+          [0, 1, 2, 3, 4].forEach((i) => {
```
Comment (Wardormeur) [inline server/routes/handlers/projects.js]: The limitation to 5 people is soft (and used to be 4, 5 was for Pioneers), only the front-end restricts to that. I'd be tempted to do a Math.max() over the length of members instead. The CSV is not paginated, so even though it's expensive, it's probably easier (in term of work length) than doing a SQL max.
As another note, a ```fields.concat(new Array(memberLength).fill(0).map((val, index) => { return member; }))``` would be more to my taste, but that's nitpicking more than anything else

File: server/routes/handlers/projects.js [hunk 2]
```diff
@@ -128,32 +128,48 @@ module.exports = {
...
-              { label: 'Supervisor Email', value: 'supervisor.email' },
-              { label: 'Owner Email', value: 'owner.email' },
-              { label: 'Created At', value: row => dateFormat(row.createdAt) },
-              { label: 'Updated At', value: row => dateFormat(row.updatedAt) },
+      return projectController.getExtended(req.query, paginated).then((projects) => {
+        const dateFormat = date => new Date(date).toLocaleDateString();
+        if (!paginated) {
+          res.setHeader('Content-Type', 'text/csv');
+          const data = projects.toJSON();
+          let fields = [
```
Comment (Wardormeur) [inline server/routes/handlers/projects.js]: fields can stay a const : we don't reassign it. const doesn't stop mutation of objects or arrays, just re-assignement.

File: server/routes/handlers/projects.js (hunk 1 above)
Comment (gr-eg) [reply server/routes/handlers/projects.js]: Updated, thanks!

File: server/routes/handlers/projects.js (hunk 2 above)
Comment (gr-eg) [reply server/routes/handlers/projects.js]: Technically I am reassigning here, https://github.com/CoderDojo/coolest-platform/pull/57/files/486ef2ed7658acd88c15a466f7dee72bb5799a2d#diff-cce739c87129ebc32c0b1a10a5c326a9R160 when I concat the arrays together.

Comment (Wardormeur) [reply server/routes/handlers/projects.js]: Ah, true. Forgot concat doesn't mutate :|

**Pull request merged**